src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -75,13 +75,108 @@ class NetworkPrivateEndpoints(ScenarioTest):
     @ResourceGroupPreparer(name_prefix='cli_test_network_private_endpoints')
     def test_network_private_endpoints(self, resource_group):
 
+        self.kwargs.update({
+            'lb': 'lb1',
+            'sku': 'Standard',
+            'vnet': 'vnet1',
+            'subnet1': 'subnet1',
+            'subnet2': 'subnet2',
+            'location': 'centralus',
+            'ip': 'pubip1',
+            'lks1': 'lks1',
+            'lks2': 'lks2',
+            'pe': 'pe1'
+        })
+
+        # Create PLS
+        self.cmd('network vnet create -g {rg} -n {vnet} --subnet-name {subnet1} -l {location}')
+        self.cmd('network lb create -g {rg} -l {location} -n {lb} --public-ip-address {ip} --sku {sku}')
+        self.cmd('network vnet subnet update -g {rg} -n {subnet1} --vnet-name {vnet} --disable-private-endpoint-network-policies')
```

Comment:
@zikalino User can just use `--disable-private-endpoint-network-policies` without `Disabled` or `Enabled`. I think `true` and `false` are also enum type.

Reply:
sorry, I didn't think straight when reading this code :-)

Reply:
😃 We can talk about this before the release. I agree this kind of usage `--disable-xxx-xxx Disabled` is a little bit strange. So we prefer just using the `--enable-xxx-xxx true` or `--enable-xxx-xxx false` right? Because for this parameter, the default value is `Enabled` and user should be able to disable it. Maybe `--enable-private-endpoint-network-policies false` is better?

Reply:
@zikalino I think ` --enable-private-endpoint-network-polices true` is better than `--disable-private-endpoint-network policies false`. However, it's a little bit complex to use `--enable-private-endpoint-network-policies false` than `--disable-private-endpoint-network-policies`. We can make a contract and I will open another PR to fix this. 😄